### ISSUE TITLE
abelfunctions on the cloud

### ISSUE THREAD

**Issue by emresertoz** [abelfunctions/abelfunctions#141]
The ultimate way to simplify installation would be to get abelfunctions on the cloud (SMC). We tried with the support team to get it running, however there were numerous errors which we could not fix (both in Sage 7.3 and in 7.4). 

If anyone is knowledgeable enough, perhaps you could fix these issues and make abelfunctions available to everyone, without installation, on the cloud. If you want to do this, I'm willing to be your guinea pig and test things out, just let me know.

In a nutshell the goal is: 
> **Get abelfunctions running on Sage Math Cloud.**

**Comment by rparini** [comment]
abelfunctions now comes installed with the "SageMath (stable)" kernel of CoCalc (SageMathCloud)
